README-idoverrideuser.md

```diff
@@ -0,0 +1,442 @@
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview without
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      description: ""
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview with internal name test_123_user
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview with internal name test_123_user
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      name: test_123_user
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview without internal name
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview without internal name
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      name: ""
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview with uid 20001
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview with uid 20001
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      uid: 20001
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview without uid
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview without uid
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      uid: ""
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview with gecos "Gecos Test"
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview with gecos "Gecos Test"
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      gecos: Gecos Test
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview without gecos
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview without gecos
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      gecos: ""
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview with gidnumber
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview with gidnumber
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      gidnumber: 20001
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview without
```

Comment:
gidnumber is missing
Example playbook to make sure test user test_user is present in idview test_idview without "gidnumber",

Reply:
Fixed.